manifests/resource/mailhost.pp

```diff
@@ -187,6 +187,24 @@
 ) {
   if ! defined(Class['nginx']) {
     fail('You must include the nginx base class before using any defined resources')
+  } elsif versioncmp($facts.get('nginx_version', $nginx::nginx_version), '1.15.0') < 0 {
+    fail('This module does not support nginx 1.14')
+  }
+  if $nginx::mail_package_name {
+    package { $nginx::mail_package_name:
+      ensure => 'installed',
+    }
+    $mail_load_content = $facts['os']['family'] ? {
+      'ArchLinux' => 'load_module /usr/lib/nginx/modules/ngx_mail_module.so;',
```

Comment:
```suggestion
      'ArchLinux' => "load_module /usr/lib/nginx/modules/ngx_mail_module.so;\n",
```

Reply:
good catch, I'll get these when I squash a bunch of the extra commits